doc/dadbod.txt

```diff
@@ -169,6 +169,9 @@ Redis ~
 Redis doesn't have a username, so use a dummy one in the URL if a password is
 required.
 
+Query parameters such as `c`, `cert`, `key`, `cacert`, `capath`, `tls-ciphers` and
```

Comment:
This line is too long. Use `textwidth=78`, as seen in the modeline.

Reply:
Corrected